code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -136,60 +136,13 @@
 
 	. = ..()
 
-// Used for an MMI or posibrain being installed into a human.
-/obj/item/organ/internal/brain/mmi_holder
-	name = "brain"
-	organ_tag = "brain"
+/obj/item/organ/internal/brain/posibrain
```

Comment:
I can't support a PR which removes the MMI interface compatibility of IPC posibrains

Reply:
Errm, You can still stick MMI's inside of an IPC and it'll work; likewise, you can still stick an IPC posibrain in an MMI and stick that in a borg (or...back in the IPC if you wanted). Other than the talking/insta-borg thing, they behave identical and solve the problem of ghosting.

Reply:
So wait, does this mean if you de-brain an IPC, you need to grab an MMI before you can put the brain back?

Reply:
That seems like fairly unintuitive behavior from the game. 🤔